app/services/analytics_events.rb

```diff
@@ -1949,10 +1950,12 @@ def idv_doc_auth_verify_proofing_results(
     step: nil,
     success: nil,
     same_address_as_id: nil,
+    app_identifier: nil,
```

Comment:
I think we should consider calling this `sp_app_identifier` (or just `sp_app_id`) to clarify that it is something coming from the SP

Reply:
sure!